### PR TITLE
Add first example to Getting-started

### PR DESCRIPTION
**Pull request by IvanPombo**
This adds a first example with Reef3d, the simplest simulator, to the Getting-started readme.

### CODE REVIEW

File: docs/Getting Started.md [hunk 1]
```diff
@@ -44,37 +44,71 @@ Next, make sure everything is set up correctly with a quick test run.
...
+|- ctrl.txt
+|- grid-000001.dat
+|- grid-000002.dat
+|- stderr.txt
+|- stdout.txt
 ```
 
+In case you want to analyze the results of the simulation, you can use the
+Paraview software.
+
```
Comment (maiyahershey) [inline docs/Getting Started.md:111]: While using the Inductiva API, keep in mind that it does not provide built-in 
visualization tools for the simulators. For such complex 3D visualizations, our team recommends open-source tools 
like [ParaView](https://www.paraview.org/download/) or [VTK](https://vtk.org/download/).

Something like this?

Comment (IvanPombo) [reply docs/Getting Started.md:112]: Made your words mine :D

File: docs/Getting Started.md [hunk 2]
```diff
@@ -44,37 +44,71 @@ Next, make sure everything is set up correctly with a quick test run.
...
+In this example, you will use the open-source hydrodynamics REEF3D simulator to simulate a [dam break scenario](https://github.com/REEF3D/REEF3D/tree/master/Tutorials/REEF3D_CFD/9_1%202D%20Dam%20Break).
 
-Running this simulation requires substantial computational resources and precise 
-configuration, challenges that have previously slowed down your project.
+With this first example, you learn that running simulations via Inductiva API is
+not much different from running them in your local machine. That's the magic
+of it all! 
 
-With Inductiva, you have the opportunity to run the dam break simulation with 
-your existing configuration files.
```
Comment (maiyahershey) [inline docs/Getting Started.md]: In this first example, you'll discover the simplicity of running simulations with the Inductiva API – it allows you to run the dam break simulation with your existing configuration files just like you would on your local machine, except it's on the Cloud. 

what do you think? (merging lines 51 till 55)

Comment (IvanPombo) [reply docs/Getting Started.md]: I think this was deprecated!

File: docs/Getting Started.md (hunk 1 above)
Comment (sarmento) [reply docs/Getting Started.md:112]: I would avoid the negative tone. Why would you have to say that you don't have something you are actually not supposed to have :)
Why not simply say that:

Now, the all the data produced by the simulator has been transfered to your local machine where you can post-process it and, in some cases, even visualize it. For example, you can visualize the contents of file X and Y using standard visualization tools suchs as A and B.

**Review by sarmento — APPROVED** [review]
Just left a comment (regarding no need to ask for forgiveness for something we didn't do), but I think it's good to go.

**Pull request merged**